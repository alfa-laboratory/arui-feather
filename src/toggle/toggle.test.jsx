import React from 'react';
import { shallow } from 'enzyme';
import Toggle from './toggle';

describe('toggle', () => {
    it('should render without problems', () => {
        let toggle = shallow(<Toggle />);

        expect(toggle).toMatchSnapshot();
    });
});
