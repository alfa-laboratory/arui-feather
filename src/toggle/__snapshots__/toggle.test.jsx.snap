// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toggle should render without problems 1`] = `
<label
  className="toggle toggle_theme_alfa-on-white"
  onBlur={[Function]}
  onFocus={[Function]}
  onMouseDown={[Function]}
  onMouseUp={[Function]}
>
  <span
    className="toggle__wrapper"
  >
    <input
      autoComplete="off"
      checked={false}
      className="toggle__control"
      onChange={[Function]}
      onClick={[Function]}
      type="checkbox"
    />
    <span
      className="toggle__switch"
    />
  </span>
</label>
`;
